packages/libs/eda/src/lib/core/hooks/computeDefaultAxisRange.ts

```diff
@@ -24,7 +24,8 @@ export function useDefaultAxisRange(
   logScale?: boolean,
   axisRangeSpec = 'Full',
   // check non-zero baseline for continuous overlay variable
-  isNonZeroBaseline: boolean = false
+  // but not used in any Viz yet, so change the default to true
+  isNonZeroBaseline: boolean = true
```

Comment:
I don't really understand this parameter. Can you add a comment explaining what "non-zero baseline" means? Thanks!

Reply:
@dmfalke That's a good question. The isNonZeroBaseline prop was intended to deal with setting a range of continuous variable for colormap legend: https://github.com/VEuPathDB/web-components/issues/454. Initially, the range was designed to be computed via the useDefaultAxisRange hook. However, I reminded myself that overall procedure was changed without using the hook, thus the prop is actually no longer used. Thus, I removed the prop and changed accordingly. To address the issue of year variable, I simply removed zero (0) case from the calculation of the min at numberDateDefaultAxisRange(). These changes are committed.

Thank you for inspiring me to recall this 👍

Reply:
Thanks for the explanation! This looks great.

Reply:
@dmfalke my pleasure 😄 Thank you for your reviews!